core/domain/prod_validators.py

```diff
@@ -755,74 +753,6 @@ def _get_external_instance_custom_validation_functions(cls):
         return [cls._validate_node_count]
 
 
-class ConfigPropertyModelValidator(base_model_validators.BaseModelValidator):
```

Comment:
Remove `config_models` from line 61 if not required anymore. (Ditto for the test file)

Reply:
Thanks for finding this. I just removed it from both files.